### DISCUSSION TITLE
Unable to create regions by dragging, when drag event starts from already played waveform area in v7

### DISCUSSION THREAD

**Discussion by cristiannoop** [katspaugh/wavesurfer.js#3295] [Q&A] [answered]
In versions < v7, wavesurfer supported the creation of regions by starting the drag event from already played waveform (the area where canvas "overlaps" with the progress element.
Since v7 this is not supported. The `.canvases` div uses `clip-path`, to hide already played waveform, thus no `pointerdown `event is emitted when clicking on the already played area and region creation is not started.

Is this intended? It would be great to still be able to create regions starting by dragging from already played area. 
Since you are able to create regions starting from not played (`.canvases`) area and dragging to the played (`.progress`) area, right-to-left, it is a counter-intuitive that you can't do it the other way around.

**Comment by katspaugh** [comment, marked as answer]
Oh, this is a regression introduced in 7.4.1 (#3285). Good catch. I'll make an issue to fix this.
